lessons/pydata/scikitlearn_api/index.ipynb

```diff
@@ -0,0 +1,1207 @@
+{
```

Comment:
<p>Nevím, jestli bych to nazval syntaxí. Spíš způsobem práce s modely v scikit-learn.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>